extensions/amp-3d-gltf/0.1/amp-3d-gltf.js

```diff
@@ -78,7 +78,7 @@ export class Amp3dGltf extends AMP.BaseElement {
 
   /** @override */
   unlayoutCallback() {
-    unobserveWithSharedInOb(this.element);
+    this.unobserveIntersections_();
```

Comment:
Several files include the optional call syntax, while others omit them. Please add in all calls to the cleanup.
```suggestion
    this.unobserveIntersections_?.();
```

Reply:
Done. FYI I originally added the optional call syntax only in cases where `observeIntersections` is called within a promise in `layoutCallback`